app/src/main/java/com/example/lapnguyen/weather_3/screen/mainscreen/MainRepositoryImplement.java

```diff
@@ -0,0 +1,28 @@
+package com.example.lapnguyen.weather_3.screen.mainscreen;
+
+/**
+ * Created by lapnguyen on 16/06/2017.
+ */
+import com.example.lapnguyen.weather_3.data.Weather;
+import java.util.List;
+import java.util.concurrent.Callable;
+import com.example.lapnguyen.weather_3.api.WeatherService;
+
+import io.reactivex.Observable;
+import io.reactivex.ObservableSource;
+import io.reactivex.annotations.NonNull;
+import io.reactivex.functions.Function;
```

Comment:
ERROR: Unused import: io.reactivex.functions.Function. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports